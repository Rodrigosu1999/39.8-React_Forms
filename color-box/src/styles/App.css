.App {
  text-align: center;
  align-items: center;
  display: flexbox;
}

