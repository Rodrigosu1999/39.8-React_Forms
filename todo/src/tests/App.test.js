import React from 'react';
import { render, screen } from '@testing-library/react';
import App from '../components/App';

it('Renders App', () => {
  render(<App />);
});

it('Passes Snapshot', () => {
  const {asFragment} = render(<App />);
  expect(asFragment()).toMatchSnapshot();
});
