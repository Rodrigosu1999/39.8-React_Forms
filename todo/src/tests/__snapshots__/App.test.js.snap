// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Passes Snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div
      class="TodoList"
    >
      <h1
        class="Todo-title"
      >
        Todo List
      </h1>
      <div
        class="Todo-items"
      >
        <div
          class="Todo"
          id="1"
        >
          <span
            class="Todo-task"
          >
            Cut my hair
          </span>
          <button>
            X
          </button>
        </div>
      </div>
      <form>
        <label
          for="task"
        >
          Task
        </label>
        <input
          id="task"
          name="task"
          type="text"
          value=""
        />
        <button>
          Add new Todo!
        </button>
      </form>
    </div>
  </div>
</DocumentFragment>
`;
